moshi-adapters-kotlinx-immutable/build.gradle.kts

```
@@ -0,0 +1,24 @@
+import org.jetbrains.kotlin.gradle.tasks.KotlinCompile
+
+plugins {
+  alias(libs.plugins.kotlinJvm)
+  alias(libs.plugins.ksp)
+  alias(libs.plugins.mavenPublish)
+}
+
+tasks.named<KotlinCompile>("compileTestKotlin") {
```

There's a DSL accessor for this

```suggestion
compileTestKotlin {
```

---

@ZacSweers 
After committing this change, the project doesn't run tests. (I commit new changes without this change)

What do you think if I revert the changes?

---

sorry should be `tasks.compileTestKotlin`. That said, it would be helpful if you elaborated on what you mean by "doesn't run tests".